examples/agent-examples.yaml

```diff
@@ -0,0 +1,76 @@
+# Example: Agents and Agent Relationships
+
+'@context':
+  https://git.io/vf-examples-jsonld-context
+  alice: https://alice.example/
+  bob: https://bob.example/
+  fablab: https://fablab.example/
+  coop: https://coop.example/
+
+'@graph':
+
+  # Agents
+
+  - '@id': alice:e1721a61-cd47-4556-84b9-8b1b81da15bf
```

Comment:
here you also need to use just `alice:` if you want to later reference that person with just `alice:`